docs/development/image_build_de.md

```diff
@@ -12,10 +12,11 @@
 
 ## 2. Start des Bauprozesses mit Packer
 
-- `cd <k8s-ecosystem-pfad>/image/`
-- `packer build -var "timestamp=$(date +%Y%m%d)" k8s-prod.json`
+- `cd <k8s-ecosystem-pfad>/image/prod/`
+- `packer init .`
+- `packer build -var "timestamp=$(date +%Y%m%d)" k8s-prod.pkr.hcl`
     - Um nur für einen bestimmten Hypervisor zu bauen, kann der `--only=`-Parameter genutzt werden
-    - Beispiel: `packer build -var "timestamp=$(date +%Y%m%d)" --only=ecosystem-virtualbox k8s-prod.json`
+    - Beispiel: `packer build -var "timestamp=$(date +%Y%m%d)" --only=ecosystem-virtualbox k8s-prod.pkr.hcl`
```

Comment:
This doesn't work any more:
```
$ packer build -var "timestamp=$(date +%Y%m%d)" --only=ecosystem-virtualbox k8s-prod.pkr.hcl
Warning: an 'only' option was passed, but not all matches were found for the given build.

Possible build names: [virtualbox-iso.ecosystem-virtualbox
vmware-iso.ecosystem-vmware].
These could also be matched with a glob pattern like: 'happycloud.*'


Error: No builds to run

A build command cannot run without at least one build to process. If the only or
except flags have been specified at run time check that at least one build is
selected for execution.
```

The command `packer build -var "timestamp=$(date +%Y%m%d)" --only=virtualbox-iso.ecosystem-virtualbox k8s-prod.pkr.hcl` seems to work, but maybe we made a mistake in the template here...